engine/src/flutter/impeller/renderer/render_target.h

```diff
@@ -177,6 +177,12 @@ class RenderTargetAllocator {
       const std::shared_ptr<Texture>& existing_color_resolve_texture = nullptr,
       const std::shared_ptr<Texture>& existing_depth_stencil_texture = nullptr);
 
+  /// @brief Disable the cache until another call to enable.
```

Comment:
This docstring talks about a cache but there is no description of anything as a cache in `RenderTargetAllocator`.  The details of `RenderTargetCache` are creeping into its superclass.  Can't we just put this into the render cache directly?  When we call it we seem to know we have a RenderTargetCache (`renderer.GetRenderTargetCache()->Enable();`)

Reply:
It is leaky, that is true. That said, if there is no cache then there is no harm in calling enable/disable.

Reply:
I think the names `RenderTargetAllocator::EnableCache` `RenderTargetAllocator::DisableCache` make more sense since you aren't disabling the allocator.

Reply:
Done